docs/evaluate/temporal-cloud/actions.mdx

```diff
@@ -30,7 +30,7 @@ The following result in an action on Temporal Cloud:
 ## Workflows
 
 - **Workflow started**.
-  Occurs via client start, client [Signal-With-Start](/sending-messages#signal-with-start), [Continue-As-New](/workflow-execution/continue-as-new), or [Child Workflow](/child-workflows) start.
+  Occurs via client start, [Continue-As-New](/workflow-execution/continue-as-new), [Child Workflow](/child-workflows) start, or for any [Signal-With-Start](/sending-messages#signal-with-start). Also occurs with [Update-With-Start](/sending-messages#update-with-start) if the Workflow was not already started.
```

Comment:
regarding Signal-with-Start: this is not wrong per se, but I see that SwS is presented to the user under the "Signals" bucket, not the "Workflows" bucket. So a user might send 10 SwS and see no billed Workflows but 10 billed Signals.